app/main.py

```diff
@@ -467,5 +467,34 @@ async def transcript(
         )
 
 
+@command_tree.command(name=command_name("query"))
+async def query(
+    interaction: discord.Interaction,
+    query: str,
+    model: gpt.OpenAIModel = gpt.OpenAIModel.GPT_4_OMNI,
+):
+    """Query a model without any context."""
+
+    await interaction.response.defer()
+
+    result = await ai_conversations.send_message_without_context(
+        bot,
+        interaction,
+        query,
+        model,
+    )
+
+    # I do not think interactions allow multiple messages.
+    messages_to_send: list[str] = []
+    if isinstance(result, Error):
+        messages_to_send = result.messages
+    else:
+        messages_to_send = result.response_messages
+
+    # I have no idea whether they actually allow you to send multiple follow-ups.
```

Comment:
i would kill u for these comment in a workplace rofl

Reply:
I think I lost it at the “completely untested” bit